extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,49 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+
+let blbSlot = {
+	config: {
+		targeting: {}
+	},
+	getSlotName: () => 'BOTTOM_LEADERBOARD',
+	getId: () => 'BOTTOM_LEADERBOARD'
+};
+
+let supportedTemplates = [ BigFancyAdBelow ];
+
+Context.extend(config);
+
+supportedTemplates.forEach((template) => {
+	TemplateService.register(template);
+});
+
+SlotService.add(blbSlot);
```

Comment:
Do we have to use SlotService and create fake blbSlot? Can't we reuse existing es5 module slotRegistry and use existing adSlots? (assuming they have the same interface)